pcsx2/GS/Renderers/HW/GSRendererNew.cpp

```diff
@@ -553,13 +553,25 @@ void GSRendererNew::EmulateBlending(bool& DATE_PRIMID, bool& DATE_BARRIER, bool&
 	// Get alpha value
 	const bool alpha_c0_zero = (m_conf.ps.blend_c == 0 && GetAlphaMinMax().max == 0);
 	const bool alpha_c0_one = (m_conf.ps.blend_c == 0 && (GetAlphaMinMax().min == 128) && (GetAlphaMinMax().max == 128));
+	const bool alpha_c0_high_min_one = (m_conf.ps.blend_c == 0 && GetAlphaMinMax().min > 128);
```

Comment:
unused

Reply:
Oh, actually I should use that one instead.

Reply:
also are you sure you don't want >= ?

Reply:
I do, good catch.